sdk/batch/azure-batch/azure/batch/models/_patch.py

```diff
@@ -0,0 +1,76 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+from typing import List, Any
+from ._models import BatchPoolReplaceContent as BatchPoolReplaceContentGenerated
+from .._model_base import rest_field
+
+__all__: List[str] = [
+    "BatchPoolReplaceContent",
+    "CreateTasksErrorException",
+]  # Add all objects you want publicly available to users at this package level
+
+class BatchPoolReplaceContent(BatchPoolReplaceContentGenerated):
+    certificate_references: List[str] = rest_field(name="certificateReferences")
```

Comment:
Is this model used as input for a request? Why doesn't the service define this property?

Reply:
`certificateReferences` is deprecated and this is a workaround for now until we're able to completely remove it. The full removal should come in our next release for 2024-07-01 I believe.